src/com/amazon/ion/benchmark/ParseAndCompareBenchmarkResults.java

```diff
@@ -1,51 +1,32 @@
 package com.amazon.ion.benchmark;
 
 import com.amazon.ion.IonDatagram;
-import com.amazon.ion.IonDecimal;
-import com.amazon.ion.IonFloat;
 import com.amazon.ion.IonList;
 import com.amazon.ion.IonLoader;
 import com.amazon.ion.IonReader;
-import com.amazon.ion.IonString;
 import com.amazon.ion.IonStruct;
 import com.amazon.ion.IonType;
-import com.amazon.ion.IonValue;
-import com.amazon.ion.IonWriter;
 import com.amazon.ion.system.IonReaderBuilder;
 import com.amazon.ion.system.IonSystemBuilder;
-import com.amazon.ion.system.IonTextWriterBuilder;
+import org.apache.commons.math3.stat.StatUtils;
+import org.apache.commons.math3.stat.descriptive.rank.Percentile;
+import org.apache.commons.math3.stat.inference.TTest;
+
 import java.io.BufferedInputStream;
-import java.io.BufferedOutputStream;
-import java.io.File;
 import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.math.BigDecimal;
-import java.math.RoundingMode;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.DoubleStream;
 
 public class ParseAndCompareBenchmarkResults {
-    public static final String RELATIVE_DIFFERENCE_SCORE = "relative_difference_score";
     public static final List<String> BENCHMARK_SCORE_KEYWORDS = Arrays.asList("speed", "Heap usage", "Serialized size", "·gc.alloc.rate");
     private static final String PRIMARY_METRIC = "primaryMetric";
-    private static final String PARAMETERS = "params";
-    private static final String INPUT = "input";
-    private static final String OPTIONS = "options";
     private static final String SECONDARY_METRIC = "secondaryMetrics";
-    private static final String SCORE = "score";
     private static final String SPEED = "speed";
-    private final static String GC_ALLOCATE = "·gc.alloc.rate";
-    private final static String HEAP_USAGE = "Heap usage";
```

Comment:
Can you explain to me why it's okay to get rid of these?

Reply:
> Can you explain to me why it's okay to get rid of these?

These two constant variables were used for composing the regression detection summary to represent which metric has regression. For now, we compose the regression detection summary by iterating the [comparisonResults](https://github.com/amazon-ion/ion-java-benchmark-cli/pull/58/commits/2b02d3a1cd79441eff09db1aa3a2ba4c23cc460e#diff-53b4790715ee277df76aa3587d5ea213712796275b4394455ea8883b508b7aafR41). The map `comparisonResults` contains key-value pairs where the keys are metrics that have regression detected and the values are the amounts they have regressed. The keys already include the names of deleted variables, so the deleted information does not need to be initialized separately.